scripts/pyxtal/compatibility_sg_n_atoms.py

```diff
@@ -0,0 +1,40 @@
+import itertools
```

Comment:
It would be nice to have at least a script-level comment to say what the goal of this script is.

Reply:
Done